docs/diagnostics/PX1035.md

```diff
@@ -5,20 +5,20 @@ This document describes the PX1035 diagnostic.
 
 | Code   | Short Description                                 | Type  | Code Fix    | 
 | ------ | ------------------------------------------------- | ----- | ----------- | 
-| PX1035 | The DAC has multiple primary key declarations | Warning (ISV Level 3: Informational) | Available | 
+| PX1035 | The DAC has multiple key declarations which use the same set of fields | Warning (ISV Level 3: Informational) | Available | 
 
 ## Diagnostic Description
-A DAC cannot have multiple primary key declarations. The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
+A DAC cannot have multiple primary, foreign, or unique key declarations which include the same fields. The warning is shown only for DACs that have the `PXCacheName` or `PXPrimaryGraph` attribute.
```

Comment:
I would say that DAC shouldn't have duplicate keys. 
**Cannot** is too strong word in this case.

Reply:
The behavior of this diagnostic was changed. Now we divide checks in two groups - the important basic checks which apply to a wider scope of DACs and more specific code style related checks which apply to a lesser scope of correctly declared DACs with `PXCacheName` or `PXPrimaryGraph` attribute.

The check for duplicate keys now applies to all non abstract DACs